test/integration/lib/philipsHue_server.js

```diff
@@ -0,0 +1,83 @@
+var express = require ('express');
+var bodyParser = require('body-parser');
+var path = require('path');
+
+var philipsHue_server = (function() {
+
+	var welcome_page = '/html/philips_initial.html';		// Philips Hub webpage
+	var suc_resp = require('./success_response.js');		// TBD: move this to appropriate module
+	var api_resp = require('./json/bridge_status.json');	// full status response
+	var light_status_resp = require('./json/lights_status.json'); // light status response
+	
+	var commanded_light_state ={1:true,2:true,3:true};		// TBD: move this to appropriate module
+	var last_received_cmd = null;		
+
+	return {
+
+		setlight: function(id,value) {  // Set the on/off status of simulated light
+			commanded_light_state[id] = value;
+		},
+
+		get last_cmd() {	
+			return last_received_cmd;
+		},
+
+		setup : function(port) {
+			var _server = express();
+			var foxbox_id = 'none';
```

Comment:
Nit: JS is usually a camel case language. We should use `foxboxId`.

Also, what value provides `'none'` compared to an `undefined` value?